# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../arphound/arphound-1.3.2-gcc43-no-alloc-header.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description:
  Do not include alloc.h because it was kept for backward compatibility until
  gcc 4.2 but got removed with the gcc 4.3 release

diff -ruN arphound-1.3.2-orig/shared/dnsquery.C arphound-1.3.2/shared/dnsquery.C
--- arphound-1.3.2-orig/shared/dnsquery.C	2006-06-03 01:26:01.000000000 +0200
+++ arphound-1.3.2/shared/dnsquery.C	2010-08-26 11:08:22.792062928 +0200
@@ -21,7 +21,6 @@
  */
 
 #include <stdio.h>
-#include <alloc.h>
 #include <time.h>
 #include <unistd.h>
 #include <string.h>
